packages/ui/src/AutoComplete/MultiAutoComplete.tsx

```diff
@@ -26,6 +26,9 @@ export type MultiAutoCompleteProps = AutoCompleteBaseProps & {
 
     /* If true, will show a loading spinner on the right side of the input. */
     loading?: boolean;
+
+    /* If true, will use `aliases` key in option item to match search value */
+    useAlias?: boolean;
```

Comment:
Let's just internally check if the `aliases` property exists and remove this flag.